cmd/transferer/main.go

```diff
@@ -37,7 +37,14 @@ func main() {
 
 	ctx := context.Background()
 
-	log.Logger = log.Output(zerolog.ConsoleWriter{Out: os.Stdout})
+	log.Logger = log.Output(zerolog.ConsoleWriter{
+		Out:     os.Stderr,
+		NoColor: true,
+		FormatTimestamp: func(ts interface{}) string {
+			t, _ := time.Parse("2006-01-02T15:04:05-07:00", ts.(string))
+			return t.Format("2006-01-02 3:04PM")
+		},
```

Comment:
you can customize time format by `TimeFormat: time.RFC3339` in console writer

Reply:
Resolved in 662bee165546790b346d873a73a42c5da480f6cd